packages/react/src/components/Storage/File-Uploader/types.ts

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { DragActionHandlers } from './hooks/useFileUploader/types';
+import { FileName } from '@aws-amplify/ui';
+
+export type SetShowPreviewer = (show: boolean) => void;
+type LevelInfo = 'public' | 'protected' | 'private';
+export type SetFileType = File[];
+
+export interface UploaderButtonProps {
+  acceptedFileTypes: Array<string>;
+  multiple?: boolean;
+  onUpload: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export interface UploaderDropProps {
+  children?: React.ReactNode;
+  inDropZone?: boolean;
+  getDropEvents?: DragActionHandlers;
```

Comment:
Rather than adding this prop we should just extend from `DragActionHandlers`

Reply:
OK, I extended that, I missed that from our API review